File: docs/_docs/code-snippets/dotnet/SqlJoinOrder.cs
using Apache.Ignite.Core.Cache.Query;

namespace dotnet_helloworld
{
    using System;
    using System.Collections.Generic;
    using Apache.Ignite.Core;
    using GridGain.Core;
    using GridGain.Core.Security;

    public static class SqlJoinOrder
    {
        public static void EnforceJoinOrder()
        {
            // tag::sqlJoinOrder[]
            var query = new SqlFieldsQuery("SELECT * FROM TABLE_A, TABLE_B USE INDEX(HASH_JOIN_IDX) WHERE TABLE_A.column1 = TABLE_B.column2")
            {
                EnforceJoinOrder = true
            };
            // end::sqlJoinOrder[]
        }
    }

}

File: docs/_docs/code-snippets/cpp/src/sql_join_order.cpp
#include <iostream>

#include "ignite/ignite.h"
#include "ignite/ignition.h"
#include "person.h"

using namespace ignite;
using namespace cache;
using namespace query;

int main()
{
	//tag::sql-join-order[]
	SqlFieldsQuery query = SqlFieldsQuery("SELECT * FROM TABLE_A, TABLE_B USE INDEX(HASH_JOIN_IDX) WHERE TABLE_A.column1 = TABLE_B.column2");
	query.SetEnforceJoinOrder(true);
	//end::sql-join-order[]
}